frontend/src/game/CurrentGames.tsx

```diff
@@ -2,42 +2,86 @@ import React, { FC, useEffect, useState } from "react";
 import Game from "./Game";
 import { getCurrentGamesAPI } from "../api/GameAPI";
 import { AxiosError } from "axios";
-import { Container, Row } from "react-bootstrap";
-import { getGameTypeString } from "../common/Utils";
+import { Button, Card, Col, Container, Row } from "react-bootstrap";
+import { getGameTypeString, mapWindToCharacter } from "../common/Utils";
+import { useNavigate } from "react-router-dom";
+import { generateJapaneseCurrentScore } from "./jp/controller/JapaneseRound";
+import { generateHongKongCurrentScore } from "./hk/controller/HongKongRound";
 
 export const CurrentGames: FC<GameTypeProp> = ({ gameVariant }) => {
+    const navigate = useNavigate();
     const [currentGames, setCurrentGames] = useState<Game[]>([]);
 
     useEffect(() => {
         getCurrentGamesAPI(gameVariant)
             .then((response) => {
-                console.log("Current games: ");
-                console.log(response.data);
                 setCurrentGames(response.data);
             })
             .catch((error: AxiosError) => {
                 alert(`Error fetching current games: ${error.response?.data}`);
             });
     }, [gameVariant]);
 
-    function getGamePlayers(game: Game) {
-        const usernames = game.players.map((player: GamePlayer) => player.username);
-        return usernames.toString();
-    }
+    const getCardHeader = (game: Game) => {
+        return (
+            <div style={{ fontWeight: "bold" }}>
+                Game {game.id} - {mapWindToCharacter(game.currentRound.roundWind)}{" "}
+                {game.currentRound.roundNumber} Bonus {game.currentRound.bonus}
+            </div>
+        );
+    };
 
-    return (
-        <Container>
-            <h1 className="my-4">Current {getGameTypeString(gameVariant)} Games</h1>
+    const getCardBody = (game: Game) => {
+        let scores: number[];
+        if (gameVariant === "jp") {
+            scores = generateJapaneseCurrentScore(game.rounds as JapaneseRound[]);
+        } else {
+            scores = generateHongKongCurrentScore(game.rounds as HongKongRound[]);
+        }
+
+        const scoresWithPlayers = scores.map((score, idx) => {
+            return {
+                username: game.players[idx].username,
+                score: score,
+            };
+        });
+        scoresWithPlayers.sort((a, b) => b.score - a.score);
```

Comment:
this seems a little confusing as the placement and the seat position both uses 1/2/3/4. Maybe just use the seat position instead

Reply:
The thing is is that mahjong soul does it this way, I can make it 1st 2nd 3rd 4th if that's better

Reply:
sounds good